src/main/java/net/minestom/server/ServerFlag.java

```diff
@@ -71,6 +71,17 @@ public final class ServerFlag {
     public static final boolean REGISTRY_LATE_REGISTER = booleanProperty("minestom.registry.late-register");
     public static final boolean REGISTRY_UNSAFE_OPS = booleanProperty("minestom.registry.unsafe-ops");
     public static final boolean EVENT_NODE_ALLOW_MULTIPLE_PARENTS = booleanProperty("minestom.event.multiple-parents");
+    /**
+     * Valid values:
+     * <ul>
+     *     <li>simple (fastest, inaccurate)</li>
```

Comment:
Maybe you could specify that this is [taxicab distance](https://en.wikipedia.org/wiki/Taxicab_geometry), just for clarity